src/nyc_trees/apps/home/views.py

```diff
@@ -56,6 +41,46 @@ def home_page(request):
     return context
 
 
+def _global_counts(past_week=False):
+    blocks_total = Blockface.objects.all().count()
+    if past_week:
+        blocks_mapped = get_block_count_past_week()
+    else:
+        blocks_mapped = Survey.objects.all().distinct('blockface_id').count()
+    if blocks_mapped > 0:
+        blocks_percent = "{:.0%}".format(
+            float(blocks_mapped) / float(blocks_total))
+    else:
+        blocks_percent = "0%"
+    if past_week:
+        blocks_percent = '+' + blocks_percent
+
+    user_count = User.objects.filter(is_active=True).count()
+
+    if past_week:
+        week_ago = now() - timedelta(days=7)
+        event_count = Event.objects \
+            .filter(begins_at__lt=now(), ends_at__gte=week_ago) \
+            .count()
+    else:
+        event_count = -999  # not needed
```

Comment:
For my education, why specify `-999` here?  If it's not needed, it would make more sense to me to simply omit it from `global_counts`